index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style.css">
+    <link rel="stylesheet" href="index.css">
+    <title>WebKit-UI</title>
+</head>
+
+<body>
+
+    <main class="main-container  ">
+
+        <div class=" nav-container pd-y-3 pd-x-3  m-auto bg-blue-5 text-color-0">
+            <div class="logo-name">
+                <h1>WebKit-UI</h1>
+            </div>
+
+            <div class="m-lf">
+
+                <ul class="wrapper">
+                    <li class="list">
+                        <a href="" class="list-inline pd-3 text-color-0">Documentation</a>
+                    </li>
+
```

Comment:
Please remove empty lines and also format the code

Reply:
only the empty line